enum.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < size
+      yield(self[i])
+      i += 1
+    end
+    self
+  end
+
+  def my_each_with_index
```

Comment:
[REQUIRED]

- [ ] my_each_with_index must return an Enumerator if no block is given.